### ISSUE TITLE
[root.cz] popup spam XMLHttpRequest

### ISSUE THREAD

**Issue by okias** [polcak/jsrestrictor#120]
```
There is a XMLHttpRequest on URL https://dpm.demdex.net/id requests...
```

while it's nice it asks, preferably it would be nicer to have it as non-blocking for loading rest of the page.

**Comment by TechnologyClassroom** [comment]
What is the purpose for demdex?  demdex.net and dpm.demdex.net are blocked on my uBlock Origin lists and Pi-hole.  I cannot reproduce this issue after whitelisting the domains on uBlock and Pi-hole.

**Comment by okias** [comment]
It's advertisement and tracking related. Not sure what features it delivers for the root.cz in this case. I also improved wording on original report, my point was to not block loading page with it waiting for

**Comment by polcak** [comment]
It seems that you are using level 3. You can either create a specific level with XML Http Requests allowed or blocked. You can install uBlock origin or other tracker blocker. In the upcoming release, we want to redesign how the wrappers are applied so it should be easier to allow a specific feature for a specific site.

**Comment by polcak** [comment]
AFAIK the modal dialog does not allow a Promise based non-blocking approach. Please let me know if I am wrong.

Please use fingerprint detector available in 0.6 instead of XHR wrappers. Please use per site modifications on allowed/blocked APIs that will land in 0.7 (see #131).

I think that this issue was partially solved in 0.6 and will be solved in 0.7 so I am closing this ticket.
